core/models.go

```diff
@@ -16,8 +16,10 @@ type VendorsModels struct {
 }
 
 func (o *VendorsModels) AddVendorModels(vendor string, models []string) {
```

Comment:
We don't need to register the DryRun Vendor so it shall not be added to this manager. Please revert this changes.